src/tls/keyexchange.h

```diff
@@ -13,6 +13,14 @@
 
 namespace tls
 {
+  inline std::string str_err(int err)
```

Comment:
We now have a shared version of this `tls::error_string` in `tls/error_string.h`, so we should use that here.

Reply:
Ah, I'd missed that, I'll fix it, thanks.